libImaging/Bands.c

```diff
@@ -51,7 +58,12 @@ ImagingGetBand(Imaging imIn, int band)
     for (y = 0; y < imIn->ysize; y++) {
         UINT8* in = (UINT8*) imIn->image[y] + band;
         UINT8* out = imOut->image8[y];
-        for (x = 0; x < imIn->xsize; x++) {
+        x = 0;
+        for (; x < imIn->xsize - 3; x += 4) {
+            *((UINT32*) (out + x)) = MAKE_UINT32(in[0], in[4], in[8], in[12]);
```

Comment:
Yes, but not just unrolling in classic sence. It also combines 4 memory sets into one and 4 increments